app_context/app_context.go

```diff
@@ -116,6 +116,19 @@ func (self *baseAppContext) SetLogger(logger logger.CtxLogger) AppContext {
 	return self
 }
 
+func (self *baseAppContext) TiltEnvironment() (string, error) {
```

Comment:
I would store this on init like everything else does... and this just returns the field from the private struct